src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -190,13 +160,57 @@ public AccordJournal start(Node node)
         Invariants.checkState(status == Status.INITIALIZED);
         this.node = node;
         status = Status.STARTING;
-        frameApplicator.start();
-        frameAggregator.start();
         journal.start();
+        this.delayedRequestProcessor.start();
         status = Status.STARTED;
         return this;
     }
 
+    @VisibleForTesting
+    public void replay()
```

Comment:
This is an extremely early version. There's a bug in flusher: we do not seem to wait for last flush before we replay, which leads to some issues, so there's sleep here. But since this patch does not hook replay up to any prod code, I thought it would be fine to leave this in.